README.rst

```diff
@@ -4,141 +4,10 @@ Open Response Assessment |build-status| |coverage-status|
-
-Clone the repo:
-
-.. code:: bash
-
-    mkdir orastack
-    cd orastack
-    git clone git@github.com:edx/edx-ora2.git
-
-Create the Vagrant instance:
-
-.. code:: bash
-
-    ln -s ./edx-ora2/Vagrantfile ./
-    vagrant plugin install vagrant-vbguest
-    vagrant up
-
-The first vagrant up will fail when setting up shared folders (because the user ora2 does not exist) so do:
-
-.. code:: bash
-
-    vagrant provision
-    vagrant reload
-
-Now you can ssh into the vagrant machine:
-
-.. code:: bash
-
-    vagrant ssh
-    sudo su ora2
-
-To install all dependencies:
-
-.. code:: bash
-
-    make install-sys-requirements
-    make install
-    make install-dev
-
-
-Running the Development Server
-==============================
-
-.. code:: bash
-
-    ./scripts/workbench.sh
-
-Additional arguments are passed to ``runserver``.  For example,
-to start the server on port 9000:
-
-.. code:: bash
-
-    ./scripts/workbench.sh 0.0.0.0:9000
-
-
-Combining and Minifying JavaScript and Sass
-============================================
-
-To reduce page size, the OpenAssessment XBlock serves combined/minified
-versions of JavaScript and CSS.  This combined/minified files are checked
-into the git repository.
-
-If you modify JavaScript or Sass, you MUST regenerate the combined/minified
-files:
-
-.. code:: bash
-
-    # Combine/minify JavaScript
-    make javascript
-
-    # Combine/minify CSS (from Sass)
-    make sass
-
-Make sure you commit the combined/minified files to the git repository!
-
-
-Running Tests
-=============
-
-To run all unit tests:
-
-.. code:: bash
-
-    make test
-
-To limit Python tests to a particular module:
-
-.. code:: bash
-
-    ./scripts/test-python.sh openassessment/xblock/test/test_openassessment.py
-
-To run just the JavaScript tests:
-
-.. code:: bash
-
-    make test-js
-
-To run the JavaScript tests in Chrome so you can use the debugger:
-
-.. code:: bash
-
-    make test-js-debug
-
-There are also acceptance and accessibility tests that run can be run against a sandbox.  For more information, about how to run these from your machine, check out `test/acceptance/README.rst <https://github.com/edx/edx-ora2/blob/master/test/acceptance/README.rst/>`__.
-
-
-i18n
-====
-
-You will need to:
-
-1. Install `i18n-tools <https://github.com/edx/i18n-tools>`_.
-2. Configure Transifex, as described in the `docs <http://docs.transifex.com/developer/client/setup>`_.
-3. Install `gettext <http://www.gnu.org/software/gettext/>`_.
-
-To extract strings and push to Transifex
-
-.. code:: bash
-
-    ./scripts/i18n-push.sh
-
-To pull strings from Transifex
-
-.. code:: bash
-
-    ./scripts/i18n-pull.sh
+Installation, Tests, and other Developer Tasks
+==============================================
 
+The previous README instructions have fallen woefully out-of-date, edX engineers now follow the guides on our `internal wiki <https://openedx.atlassian.net/wiki/display/TNL/ORA+FAQ>`_.
```

Comment:
I don't have a problem linking to the wiki, though I would not refer to "internal wiki" since it is public. I find it easier to update the wiki (and more likely that people will actually do so) then to require all changes to go through a PR.